build.gradle

```diff
@@ -1,11 +1,33 @@
-apply plugin: 'java'
+buildscript {
+    ext.kotlin_version = '1.3.61'
 
-group = pluginGroup
-version = pluginVersion
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.github.jengelman.gradle.plugins:shadow:5.2.0'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'idea'
+}
+
+apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+//Put your plugins folder directory here! If it is left as '', jars will only be copied if the parent project has a path specified
+ext.plugin_path = ''
+
+group = 'com.derongan.minecraft'
```

Comment:
Can these changes be broken into a different review? I want to accept them before I go over the kt stuff

Reply:
Is this resolved in master now? If so, I'll resolve when I fix conflicts.